common/tools/eslint-plugin-azure-sdk/src/rules/ts-package-json-files-required.ts

```diff
@@ -46,37 +72,54 @@ export = {
 
             const nodeValue = node.value;
             const elements = nodeValue.elements as Literal[];
-            const elementValues = elements.map((element: Literal): unknown => element.value);
+            let elementValues = elements.map((element: Literal): unknown => element.value);
 
-            // looks for 'dist' with optional leading './' and optional trailing '/'
-            if (
-              elements.every(
-                (element: Literal): boolean =>
-                  !/^(.\/)?((dist\/)|(dist$))/.test(element.value as string)
-              )
-            ) {
-              context.report({
-                node: nodeValue,
-                message: "dist is not included in files",
-                fix: (fixer: Rule.RuleFixer): Rule.Fix => {
-                  elementValues.push("dist");
-                  return fixer.replaceText(nodeValue, arrayToString(elementValues));
+            let currBadPatterns: string[] = [];
+            let currRequiredPatterns = [...requiredPatterns];
+            const fullMatchIndex = 0;
+            const patternRootMatchIndex = 1;
+            elements.forEach((element) => {
```

Comment:
Any reason not to use `for (const element of elements)` instead of `elements.forEach` here? IMO it's more readable to use the iterative style rather than `forEach`.

Reply:
for/of is recommended for readability

Reply:
it was this way from before and I kept it, I changed it to `for/of`.